src/lib/table/cell.ts

```diff
@@ -121,7 +121,7 @@ export class MatFooterCell extends CdkFooterCell {
 })
 export class MatCell extends CdkCell {
   constructor(columnDef: CdkColumnDef,
-              elementRef: ElementRef) {
+              elementRef: ElementRef<HTMLElement>) {
```

Comment:
`<HTMLElement | HTMLTableCellElement>`

Reply:
Never mind, `type1 | type2` doesn't give you anything if `type2` extends `type1`.